x-pack/plugin/esql/qa/testFixtures/src/main/resources/eval.csv-spec

```diff
@@ -358,7 +358,7 @@ docsEvalUnnamedColumnStats
 // tag::evalUnnamedColumnStats[]
 FROM employees
 | EVAL height * 3.281
-| STATS avg_height_feet = AVG(`height*3.281`)
+| STATS avg_height_feet = AVG(`height * 3.281`)
```

Comment:
CSV test that showed the previous bug - instead of matching on the name, the * acted as a wildcard.